### PR TITLE
Add support for env variables

### PR DESCRIPTION
**Pull request by zhenik**
@fredrikhgrelland @m88614  what do you think about this approach?

### CODE REVIEW

File: ansible/startup.yml [hunk 1]
```diff
@@ -7,6 +7,13 @@
       - consul
       - nomad
   tasks:
+    - name: "Verify default env variables"
+      set_fact:
+        var_consul_acl: "{{ lookup('env', 'VAR_CONSUL_ACL') | default('allow', true) }}"
+        var_environment: "{{ lookup('env', 'VAR_ENVIRONMENT') | default('dev', true) }}"
```
[the remark this answers is not in the document]
Comment (zhenik) [reply ansible/startup.yml:13]: Setup Ansible vars from env, if variable env variable is not defined ~> setup default value.


## WHY?

In the example `test/Vagrantfile` part 
```
# override env variables
config.vm.provision "file", source: "custom-env.sh", destination: "/home/vagrant/custom-env.sh"
config.vm.provision :shell, inline: "echo 'source /home/vagrant/custom-env.sh' > /etc/profile.d/custom-env.sh", :run => 'always'
```
is **OPTIONAL**.

[the remark this answers is not in the document]
Comment (fredrikhgrelland) [reply ansible/startup.yml:13]: I see. So it really comes down to having really good defaults.

[the remark this answers is not in the document]
Comment (fredrikhgrelland) [reply ansible/startup.yml:13]: Maybe if you just write up the documentation in this PR we can call it? This comes down to good documentation on what it does out of the box and what you must do to change it. If we can pre-package a set of different flavors of configurations that would be super. We could use #82 to use one or the other? or maybe that can sort the override part? Moving it from vagrant to ansible in a `pre-startup.yml`-step?

**Comment by zhenik** [comment]
@fredrikhgrelland  @m88614 So far, the current approach does not seem to me as a good solution. I will provide more approaches in issue #75 , let us keep the discussion there

**Pull request closed without merging**